assisted_deployment.sh

```diff
@@ -7,18 +7,232 @@ source common.sh
 source assisted_deployment_utils.sh
 
 
+ASSISTED_SERVICE_SERVICE_IMAGE="${ASSISTED_SERVICE_SERVICE_IMAGE:-}"
```

Comment:
Will document before removing WIP :)

Reply:
Maybe it's enough to just make it `ASSISTED_*`?